src/profile/components/emailEditor/EmailEditor.tsx

```diff
@@ -212,6 +212,9 @@ function EmailEditor(): React.ReactElement | null {
           {t('profileForm.addEmail')}
         </Button>
       )}
+      {!willSendEmailVerificationCode && (
+        <EditingNotifications content={content} dataType={dataType} />
+      )}
```

Comment:
This is good for now. Just a note for future: this will need changes once the username/password authentication option will be taken into use. The email verification will happen for any user who has logged in via Tunnistus, no matter if it was using suomi.fi or some other means.

Reply:
Ok, that is a good change.